plugin/pprof/README.md

```diff
@@ -42,11 +46,11 @@ Listen on an alternate address:
 }
 ~~~
 
-Listen on an all addresses on port 6060:
+Listen on an all addresses on port 6060: and enable block profiling
 
 ~~~ txt
 . {
-    pprof :6060
+    pprof :6060 1
```

Comment:
lgtm, but I would suggest making this
~~~
pprof ADDRESS {
    blockrate [RATE]
}
~~~
where the default will be 1 is just `blockrate` is specified.

This change helps if we want even more options later.

Reply:
ok updated.
```
pprof ADDRESS {
    block [RATE]
}
```